examples/platform/silabs/efr32/rs911x/sl_wifi_if.c

```diff
@@ -0,0 +1,754 @@
+/*
```

Comment:
This looks like a highly duplicated file from `examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.c`

why can't they be merged ?